app/renderer/js/utils/link-util.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const wurl = require('wurl');
+
+let instance = null;
+
+class LinkUtil {
+	constructor() {
+		if (instance) {
+			return instance;
+		} else {
+			instance = this;
+		}
+
+		return instance;
+	}
+
+	isInternal(currentUrl, newUrl) {
+		const currentDomain = wurl('hostname', currentUrl);
+		const newDomain = wurl('hostname', newUrl);
+
+		const skipImages = '.jpg|.gif|.png|.jpeg|.JPG|.PNG';
+
+		// We'll be needing this to open images in default browser
+		return (currentDomain === newDomain) || newUrl.match(skipImages);
```

Comment:
This should be -
```javascript
return (currentDomain === newDomain) && !newUrl.match(skipImages);
```
We don't want to open the image link within the app; it should be opened in the default browser.